src/modules/html_functions.js

```diff
@@ -0,0 +1,121 @@
+const tasksList = document.getElementById('tasks-list');
+const popup = document.getElementById('popup');
+
+export const showPopup = () => {
+  popup.style.display = 'block';
+  setTimeout(() => {
+    popup.style.display = 'none';
+  }, 3000);
+};
+
+const doneButtonListener = (toDoList, taskItem, taskInput,
+  task, doneButton, removeButton, moreButton) => {
+  if (toDoList.taskExists(taskInput.value, task.index)) {
+    showPopup();
+    return;
+  }
+
+  taskInput.disabled = true;
+  task.description = taskInput.value;
+  toDoList.updateLocalStorage();
+
+  doneButton.remove();
+  removeButton.remove();
+  taskItem.appendChild(moreButton);
```

Comment:
- [ ] You're currently missing the code to update the localstorage of the `completed` flag upon completing a task. When a user clicks on the checkbox to mark a task as completed, the localstorage should reflect that change, but at the moment it only stays on `false`. Please make sure to address this issue, as per requirement..

Reply:
According to my understanding, We are supposed to implement this change tomorrow, when we add an event listener to the Check box.

This is the task for tomorrow: [link](https://github.com/microverseinc/curriculum-javascript/blob/main/todo-list/m2_interactive_list.md)

When I add the event Listener to the Check Box and update the checked value, the updated value will automatically be saved to local storage.

Not that you are ever gonna read this @ReshyResh  but it would save us a lot of time if You guys can check the project requirements first before requesting changes.